client/my-sites/plans/plan-feature/google-analytics.js

```diff
@@ -15,134 +15,110 @@ import PlanCard from 'my-sites/plan-card';
 
-	const freeTitle = i18n.translate( 'Free' );
-	const freeLine = i18n.translate( 'Free for life' );
-	const freeButton = i18n.translate( 'Your Plan' );
-
-	const premiumTitle = i18n.translate( 'Premium' );
-	const premiumLine = i18n.translate( '$99 per year' );
-	const premiumButton = i18n.translate( 'Your Plan' );
-
-	const businessTitle = i18n.translate( 'Business' );
-	const businessLine = i18n.translate( '$299 per year' );
-	const businessButton = i18n.translate( 'Upgrade' );
-
-	const isPremium = site && site.plan && site.plan.product_name_short === 'Premium';
-
 	function goBack() {
-		page.back( '/plans/' + ( context.params.domain || '' ) );
+		const fallback = '/plans/' + ( context.params.domain || '' );
```

Comment:
Instead of the if else, we should be able to simplify this to:
```
page.back( fallback );
```
Page.js navigates to your fallback link if your history is empty:
https://github.com/visionmedia/page.js/blob/master/page.js#L224